CHANGELOG.md

```diff
@@ -8,6 +8,9 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 - civis.io functions use the "hidden" API option to keep jobs out of the UI. Deprecate the "archive" parameter in favor of "hidden".
 - civis.io.query_civis now has a "hidden" parameter which defaults to True
 
+### Performance Enhancements
+- Decrease time required to create client objects from ~0.6 seconds to ~150 us for all objects after the first in a session
```

Comment:
It would have been if I knew how to type Greek letters in Github-markdown. Or is unicode okay in these files?